python/solver.py

```diff
@@ -518,7 +516,8 @@ def _output_vector_field(self, curfield_type, fname_prefix, output_k, component)
             f['Bloch wavevector'] = np.array(output_k)
             self._write_lattice_vectors(f)
 
-            self.mode_solver.multiply_bloch_phase()
+            if curfield_type != 'v':
+                self.mode_solver.multiply_bloch_phase()
```

Comment:
Right, this corresponds to https://github.com/stevengj/mpb/blob/0fa695b03b642c423b5a1b262c2804d2418f62ef/mpb/fields.c#L714-L723